src/serviceorchestration/5_0_0/sd/service-orchestration-lock-management/service_orchestration_lock_management_sd.tex

```diff
@@ -0,0 +1,332 @@
+
+It also can happen that a certain service instance is reserved (locked) upon a consumer system request, but for any reason it needs to be released. In this case a higher entity can search for the associated orchestration lock record by consuming the \textit{query} service operation in order to check the expiration time and the owner of the lock. If the service instance is need to be released before the expiration time of the lock, then the \textit{remove} service operation can be consumed.
+
+\subsection{Important Delimitations}
+\label{sec:delimitations}
+
+The requester has to identify itself to use any of the operations. 
+
+\subsection{Access policy}
+\label{sec:accesspolicy}
+
+The service is only available for operators, dedicated Core/Support systems and those who have the proper authorization rights to consume it.
+
+\newpage
+
+\section{Service Operations}
+\label{sec:functions}
+
+This section describes the abstract signatures of each operations of the service. The \textbf{serviceOrchestrationLockManagement} service is used to \textit{create}, \textit{remove} and \textit{query} orchestration lock records.
+In particular, each subsection names an operation, an input type and one or two output types (unsuccessful operations can return different structure), in that order.
+The input type is named inside parentheses, while the output type is preceded by a colon. If the operation has two output types, they are separated by a slash.
+Input and output types are only denoted when accepted or returned, respectively, by the operation in question. All abstract data types named in this section are defined in Section 3.
+
+\phantomsection
+\fsubsection{create}{OrchestrationLockListRequest}{OrchestrationLockListResponse}{}{ErrorResponse}
+
+Operation \textit{create} creates orchestration lock records in bulk. Each lock requires a service instance identifier, the owner system name and an expiration time. 
+
+\fsubsection{query}{OrchestrationLockQueryRequest}{OrchestrationLockListResponse}{}{ErrorResponse}
+
+Operation \textit{query} lists the orchestration lock records that match the filtering requirements. The query data must meet the following criteria:
+
+\begin{itemize}
+    \item The operation returns results in pages. There are default page data settings, but the requester can provide a custom specification.
+    \item If page number is specified, the page size must be specified as well and vice versa.
+    \item In some Local Clouds there is a maximum page size.
+    \item If a filter expects a list, there is an OR relation between the elements of the filter.
+    \item There is an AND relation between different kind of filters.
+\end{itemize}
+
+\fsubsection{remove}{OrchestrationLockRemoveRequest}{}{OperationStatus}{ErrorResponse}
+
+Operation \textit{remove} deletes the orchestration lock records by owner system name and service instance identifiers.
+
+\clearpage
+
+\section{Information Model}
```

Comment:
Ordering should be the following:

1. OrchestrationLockListRequest
2. Identity
3. OrchestrationLockRequest
4. OrchestrationLockListResponse
5. OrchestrationLockResponse
6. ErrorResponse
7. OrchestrationLockQueryRequest
8. OrchestrationLockRemoveRequest